webcompat/static/css/src/nav.css

```diff
@@ -120,4 +120,66 @@
   }
 }
 
-/* add icons to nav, right + left with x1-fix : x2, pseudo-links, sticky nav */
+
+.nav-button {
+    border: none;
```

Comment:
nit: we seem to go between 2 spaces and 4 spaces for indentation.

@zoepage do you have editorconfig set up for your texteditor?

https://github.com/webcompat/webcompat.com/blob/master/.editorconfig#L14

Reply:
🙈 sorry. Will take care of it.